forceng.js

```diff
@@ -192,8 +197,14 @@ angular.module('forceng', [])
         if (params.instanceURL) {
           if (!oauth) oauth = {};
           oauth.instance_url = params.instanceURL;
+        }else if(params.instanceUrl) {
+          if (!oauth) oauth = {};
+          oauth.instance_url = params.instanceUrl;            
+        }else if(params.instance_url) {
+          if (!oauth) oauth = {};
+          oauth.instance_url = params.instanceURL;            
```

Comment:
Very good catch Matt. I wonder how come it's not an issue, its been like that for a long time.